test-definitions/fern/apis/query-parameters/definition/user.yml

```diff
@@ -0,0 +1,30 @@
+types:
+  User:
+    properties:
+      name: string
+      tags: list<string>
```

Comment:
should we add tests for nested objects + maps as well?

Reply:
Nested objects have tests below! I can include for the map type though, good call.

Reply:
Realized I thought you were commenting in the `fern check` tests earlier - added a nested object and a `map` here to the test definition here!